draft-ietf-rats-architecture.md

```diff
@@ -498,10 +498,10 @@ For example, a carrier-grade router consists of a chassis and multiple slots.
 The trustworthiness of the router depends on all its slots' trustworthiness.
 Each slot has an Attesting Environment such as a TEE collecting the
 Claims of its boot process, after which it generates Evidence from the Claims.
-Among these slots, only a main slot can communicate with the Verifier
-while other slots cannot. But other slots can communicate with the main
-slot by the links between them inside the router. So the main slot collects
-the Evidence of other slots, produces the final Evidence of the whole router and
+Among these slots, only a "main" slot can communicate with the Verifier
+while other slots cannot, but other slots can communicate with the main
+slot across the links between slots inside the router. The main slot thus collects
```

Comment:
```suggestion
while other slots cannot, but other slots can communicate with the "main"
slot across the links between slots inside the router. The "main" slot thus collects
```

Reply:
I'm not sure why "main" is quoted, but should it be consistent?

Reply:
It was quoted the first time as being defined in context, and then used unquoted thereafter.  See for example https://blog.apastyle.org/apastyle/2013/02/quotation-mark-uses-other-than-quotes.html which quotes the guidance:
```
To introduce a word or phrase used as an ironic comment, as slang, or as an invented or coined expression. Use quotation marks the first time the word or phrase is used; thereafter, do not use quotation marks. (p. 91)
```
(Here "main" router is an invented or coined expression)